packages/notte-browser/src/notte_browser/session.py

```
@@ -60,6 +59,10 @@ class TrajectoryStep(BaseModel):
     action: BaseAction
 
 
+class LocalStepRequestDict(StepRequestDict):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)